docs/src/HowToGuides/Atmos/Microphysics.md

```diff
@@ -132,9 +286,30 @@ where:
  - ``q_{liq}`` - liquid water specific humidity,
  - ``\tau_{cond\_evap}`` - relaxation timescale (parameter in `MicrophysicsParameters` module).
 
-## Autoconversion
+### Cloud ice sublimation/resublimation
```

Comment:
I think this should be called "deposition." And I would switch the order to match "condensation/evaporation" where you have the one that increases size first and then the one that decreases size.
```suggestion
### Cloud ice deposition/sublimation
```